lambdas/scripts/upload_ecr.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+error() {
+    echo $@ 2>&1
+    exit 1
+}
+
+[[ $# == 2 ]] || error "Usage: $0 account_id image_name"
+
+account_id=$1
+image_name=$2
+
+regions=$(aws ec2 describe-regions --query "Regions[].{Name:RegionName}" --output text)
+
+for region in $regions
```

Comment:
maybe we should use pre-defined list of regions, so our scripts won't fail when new regions are added?

Reply:
I don't actually know what's worse. If new regions are added, we probably _should_ support them immediately? CI failing is not great, but CI succeeding and then customer deployments failing is even worse - so maybe it's good to fail here so it's obvious.

Reply:
> we probably should support them immediately?

well, at the moment "support" means skipping a region in a script 😁

Reply:
failing job is OK but we need either
1. provide clear steps how to add a new region (preferred option)
2. deployments to old regions should still work

Reply:
2 sounds almost impossible: we either a) ignore a failed upload, let the job succeed, and hide the problem (bad), or b) not return an error immediately, allow subsequent CI steps to run (upload to GovCloud, etc.), but somehow return an error at the very end. But even if we do b), what guarantees do we have? Maybe there was an unrelated network failure in addition to the new region, but it goes unnoticed.

So anyways, I'd rather do 1, but add the instructions once we actually have a good way to configure new regions (TF, CloudFormation, whatever).

Reply:
> So anyways, I'd rather do 1, but add the instructions once we actually have a good way to configure new regions (TF, CloudFormation, whatever).

🆗